### PR TITLE
add qna notifications key

### PR DESCRIPTION
**Pull request by dannimajdalani**
related https://github.com/vanilla/vanilla-cloud/pull/331/
related https://github.com/vanilla/vanilla-cloud/issues/363
### Keys added
'Send me follow up emails for my answered questions.'
'This user has already received a notification within the allowed time frame.'

### CODE REVIEW

File: tx-source/site_core.php [hunk 1]
```diff
@@ -1581,7 +1589,9 @@
 $Definition['This Month\'s Leaders'] = 'This Month\'s Leaders';
 $Definition['This setting loads the page before executing Javascript which can improve your SEO.<br><strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong>'] = 'This setting loads the page before executing Javascript which can improve your SEO.<br><strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong>';
 $Definition['This question\'s answer has been accepted.'] = 'This question\'s answer has been accepted.';
+$Definition['A follow-up email was already sent.'] = 'A follow-up email was already sent.';
```
Comment (tburry) [inline tx-source/site_core.php:1592]: This "follow-up" has a dash where we don't use a dash anywhere else. It's important to use consistent phrasing.

File: tx-source/site_core.php [hunk 2]
```diff
@@ -1581,7 +1589,9 @@
...
 $Definition['This user has been verified as a non-spammer.'] = 'This user has been verified as a non-spammer.';
+$Definition['This user has disabled this notification preference.'] = 'This user has disabled this notification preference.';
```
Comment (tburry) [inline tx-source/site_core.php:1594]:
```suggestion
$Definition['The user has disabled this notification preference.'] = 'The user has disabled this notification preference.';
```

I don't like sentences that specify "this" twice.

File: tx-source/site_core.php [hunk 3]
```diff
@@ -1236,6 +1237,11 @@
 $Definition['Q&A Question'] = 'Question';
 $Definition['QnA Accepted Answer'] = 'Answer ✓';
 $Definition['QnA Rejected Answer'] = 'Rejected Answer';
+$Definition['QnAFollowUp.Email.Message'] = 'We noticed you have at least one answer to your question. Did this resolve your question or do you still need help?
+<p>If your question has been resolved please visit the link below and mark one or more answers as <b>accepted</b>
+<p>This will help others who may have the same issue to easily find a solution.';
```
Comment (tburry) [inline tx-source/site_core.php:1242]:
```
<p>We noticed you have at least one answer to your question. Can you visit the community and see if any of the answers resolve your question?</p>
<p>If you see an answer you find helpful, please accept one of the answers.</p>
```

File: tx-source/dash_core.php [hunk 4]
```diff
@@ -756,6 +759,7 @@
 $Definition['This category is archived.'] = 'This category is archived.';
 $Definition['This connection is in test-mode.'] = 'This connection is in test-mode.';
 $Definition['This Database'] = 'This Database';
+$Definition['This feature trigger emails to authors of answered questions that have not an accepted answer yet.'] = 'This feature trigger emails to authors of answered questions that have not an accepted answer yet.';
```
Comment (tburry) [inline tx-source/dash_core.php:762]: Make this a smaller variable code and change the translation to:

```php
'This feature sends emails to the authors of answered questions that do not have an accepted answer.'
```

File: tx-source/dash_core.php [hunk 5]
```diff
@@ -326,6 +326,7 @@
 $Definition['Enable Pocket Locations'] = 'Enable Pocket Locations';
 $Definition['Enable Private Communities'] = 'Enable Private Communities';
 $Definition['Enable Product Integration'] = 'Enable Product Integration';
+$Definition['Enable follow up notifications.'] = 'Enable follow up notifications.';
```
Comment (tburry) [inline tx-source/dash_core.php:329]: Notice the two strings above use title case without a period, while you use sentence case.

Comment (igraziatto) [reply tx-source/dash_core.php]: Those strings above are related to other places of the Dashboard, not the categories add/edit:

Screen Shot 2020-08-06 at 2 04 36 PM

I was following the pattern specific to this page.

**Pull request merged**